src/contact-form.js

```diff
@@ -190,9 +191,15 @@ class ContactForm {
         .done(() => {
           this.setSubmitButtonState('success');
           this.cleanElementsValue();
+          if(mouseflow){
+            mouseflow.formSubmitSuccess(formRoot);
+          };
           successCallback(metricsData);
         })
         .fail(() => {
+          if(mouseflow){
+            mouseflow.formSubmitFailure(formRoot);
+          };
```

Comment:
Can you make this inline please?
`if (mouseflow) mouseflow.formSubmitFailure(formRoot);`